src/drafts/Skeleton/_BaseSkeletonBone.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import styled, {keyframes} from 'styled-components'
+import sx, {SxProp} from '../../sx'
+import {get} from '../../constants'
+
+type BaseSkeletonBoneProps = {
+  height?: React.CSSProperties['height']
+  width?: React.CSSProperties['width']
+} & SxProp
+
+const shimmer = keyframes`
+  from { mask-position: 200%; }
+  to { mask-position: 0%; }
+`
+
+export const BaseSkeletonBone = styled.div<BaseSkeletonBoneProps>`
+  animation: ${shimmer};
+  display: block;
+  background-color: var(--bgColor-muted, ${get('colors.canvas.subtle')});
```

Comment:
It's possible that somebody could render a skeleton loader on top of something with the `bgColor-muted` background color. This would make the loader invisible except for the "shimmer" animation.

Reviewers, how do you think we should reckon with this?
1. Add a boolean prop to change the color of a skeleton loaders. Kind of like the `contrast` prop on TextInput
2. Add a component-specific color token that uses another color from the [base gray scale](https://primer.style/primitives/storybook/?path=/story/color-base-scales--gray).
3. Keep this as-is until we identify a real use-case where this causes a problem.